static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -161,6 +162,15 @@ const getWhitelistedQueryParams = (): {} => {
     return pick(getUrlVars(), whiteList);
 };
 
+const getUrlKeywords = (pageId: string): string => {
+    if (pageId) {
+        const segments = pageId.split('/');
+        const lastPathname = segments.pop() || segments.pop(); // This handles a trailing slash
```

Comment:
I don't understand this, why pop twice if the first didn't result on anything? Has this something to do with empty space?

Reply:
This should, if I'm correct mean that something like `root/hello-world/` will return the correct value of `hello-world`. This is because the first pop will remove the empty space that's at the end of the array.
```javascript
const hello = '/root/hello-world/'
const segments = hello.split('/') // ["", "root", "helllo-world", ""]
segments.pop() // Result: ""
segments.pop() || segments.pop() // Result: "hello-world"
```

We might never have a trailing slash, but I figured it was safer to include